go/cmd/vtgate/cli/status.go

```diff
@@ -37,7 +37,10 @@ func addStatusParts(vtg *vtgate.VTGate) {
 	servenv.AddStatusPart("Gateway Status", vtgate.StatusTemplate, func() any {
 		return vtg.GetGatewayCacheStatus()
 	})
-	servenv.AddStatusPart("Health Check Cache", discovery.HealthCheckTemplate, func() any {
+	servenv.AddStatusPart("Health Check - Cache", discovery.HealthCheckCacheTemplate, func() any {
 		return vtg.Gateway().TabletsCacheStatus()
 	})
+	servenv.AddStatusPart("Health Check - Healthy Tablets", discovery.HealthCheckHealthyTemplate, func() any {
```

Comment:
here we have "Healthy Tablets" but in the vtcombo version it is just "Healthy". i think we should be consistent between the two, but i have no preference which one you standardize on

Reply:
I would even suggest dropping "Tablet" from the table headers. Hope no one is depending on the header values though :)

Reply:
Good catch thank you!

Reply:
Fixed via f6d526e8da